docs/marketplace/dev/setup/infrastructure-setup.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Infrastructure setup
+description: This article provides required system infrastructure setup for a Marketplace project.
+template: concept-topic-template
+---
+
+This article provides required system infrastructure setup for a Marketplace project.
+
+## Applications resources visibility
+
+| Service                          | Resources available               | exposed to WAN? |
+| -------------------------------- | --------------------------------- | ---------------- |
+| MerchantPortal                                | PrimaryDatabase, QueueBroker, MerchantPortalSessionStorage | yes
+| Backoffice                                | PrimaryDatabase, QueueBroker, BackofficeSessionStorage | no, accessed through VPN
+| Scheduler                                | PrimaryDatabase, QueueBroker, Storage, Search | no, accessed through a Bastion
+| REST API (Glue)                                | Storage, Search, Gateway | yes
+| Storefront (Yves)                                | Storage, Search, Gateway | yes
+
+The following diagram shows the recommended infrastructure setup for a Marketplace Spryker cluster:
+
+![Entity diagram](https://confluence-connect.gliffy.net/embed/image/d10f1622-0c0e-44f0-b4f6-46d294448dec.png?utm_medium=live&utm_source=custom)
```

Comment:
please update the diagram to reflect customer bootstrap for MerchantPortal (on diagram it is pub/Zed/index.php.)

Reply:
fixed